fast-af/fast-checkout-woocommerce
# PINT-801 - Add users for test mode

# Description

Add an option to add users who can view the Fast buttons in Test Mode.

# Testing instructions

1. Login to staging.
2. Go to the Fast settings page and click the Test mode tab.
3. On the Test Mode Users option, start typing the name of a user who is not an administrator.
4. Select that user and save the changes.
5. Enable Test mode.
6. Verify that the selected user can still view the buttons.

# Checks
